src/tests/modify.test.js

```diff
@@ -396,4 +396,44 @@ describe('modify() - basic mutations', () => {
       },
     });
   });
+
+  it('customize x-jsf-presentation (shorthand)', () => {
+    const result = modify(schemaPet, {
+      fields: {
+        pet_age: () => {
+          return {
+            presentation: {
+              'data-foo': 123,
+            },
+          };
+        },
+        'pet_address.street': {
+          errorMessage: {
```

Comment:
This test was not correct, I fixed it directly in another PR as hotfix https://github.com/remoteoss/json-schema-form/pull/79/commits/76bb7651b4ee4e9a898f34321090bf5a03b20c53. (Thank you @danielcardoso5 for spotting it 🦅 )